source/linux/Foundational_Components/Kernel/Kernel_Drivers/Storage/MMC-SD.rst

```diff
@@ -294,160 +315,236 @@ Driver Configuration
+         no-1-8-v; /* disabling all the UHS modes */
+      };
 
-          ::
-
-              &sdhci1 {
-                  /* SD/MMC */
-                  vmmc-supply = <&vdd_mmc1>;
-                  vqmmc-supply = <&vdd_sd_dv>;
-                  pinctrl-names = "default";
-                  pinctrl-0 = <&main_mmc1_pins_default>;
-                  ti,driver-strength-ohm = <50>;
-                  disable-wp;
-                  no-1-8-v; /* disabling all the UHS modes */
-              };
+#. Reduce the bus width
 
+   The SD interface supports a bus width of 4. It can be reduced to 1 by
+   changing the **bus-width** device tree property from 4 to 1.
 
+   .. code-block:: diff
 
-#. Reduce the bus width
+      diff --git a/arch/arm64/boot/dts/ti/k3-am62-main.dtsi b/arch/arm64/boot/dts/ti/k3-am62-main.dtsi
+      index 7bbfcb158842..2ef974f7206f 100644
+      --- a/arch/arm64/boot/dts/ti/k3-am62-main.dtsi
+      +++ b/arch/arm64/boot/dts/ti/k3-am62-main.dtsi
+      @@ -424,7 +424,7 @@
+         ti,itap-del-sel-sdr12 = <0x0>;
+         ti,itap-del-sel-sdr25 = <0x0>;
+         ti,clkbuf-sel = <0x7>;
+      -     bus-width = <4>;
+      +     bus-width = <1>;
+      };
 
-  - The SD interface supports a bus width of 4. It can be reduced to 1 by
-    changing the ``bus-width`` device tree property from 4 to 1.
+      sdhci2: mmc@fa20000 {
 
-      ::
-
-          diff --git a/arch/arm64/boot/dts/ti/k3-am62-main.dtsi b/arch/arm64/boot/dts/ti/k3-am62-main.dtsi
-          index 7bbfcb158842..2ef974f7206f 100644
-          --- a/arch/arm64/boot/dts/ti/k3-am62-main.dtsi
-          +++ b/arch/arm64/boot/dts/ti/k3-am62-main.dtsi
-          @@ -424,7 +424,7 @@
-                          ti,itap-del-sel-sdr12 = <0x0>;
-                          ti,itap-del-sel-sdr25 = <0x0>;
-                          ti,clkbuf-sel = <0x7>;
-          -               bus-width = <4>;
-          +               bus-width = <1>;
-           };
+|
 
-           sdhci2: mmc@fa20000 {
+.. _create-partitions-in-emmc-uda-from-linux:
+
+Create software partitions in eMMC UDA
+**************************************
+
+In eMMC, the User Data Area (UDA) HW partition is the primary storage
+space generally used to flash the rootfs. To prepare the UDA, use
+the :command:`fdisk /dev/mmcblkn` command in which 'n' is 0 or 1.
+To find which integer is eMMC use the command :command:`lsblk`:
+
+.. code-block:: console
+
+   root@<machine>:~# lsblk
+   NAME         MAJ:MIN RM  SIZE RO TYPE MOUNTPOINTS
+   mmcblk0      179:0    0 14.8G  0 disk
+   mmcblk0boot0 179:32   0 31.5M  1 disk
+   mmcblk0boot1 179:64   0 31.5M  1 disk
+   mmcblk1      179:96   0 14.8G  0 disk
+   |-mmcblk1p1  179:97   0  128M  0 part /run/media/boot-mmcblk1p1
+   `-mmcblk1p2  179:98   0  1.9G  0 part /
+
+Where the eMMC will have hardware partitions :file:`mmcblknboot0`
+and :file:`mmcblknboot1`. The :file:`mmcblkn` is the eMMC device.
+
+Now we use :command:`fdisk /dev/mmcblk0` to create one software partition
```

Comment:
```suggestion
Now we use ``fdisk /dev/mmcblk0`` to create one software partition
```

Command should be used for introducing command line tools. For example commands you should either use inline code like about or the sample role:

https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#role-samp